config/environment_indicator.indicator.yml

```diff
@@ -0,0 +1,3 @@
+name: localllllll
```

Comment:
whoops this shouldn't have made it in here.

Reply:
removed